MAUI/DataGrid/Column-types.md

```diff
@@ -342,7 +342,7 @@ The `DataGridCheckBoxColumn` inherits all the properties of the `DataGridColumn`
 {% endhighlight %}
 {% endtabs %}
 
-![DataGrid with CheckBox column](Images\column-types\maui-datagrid-column-checkbox.png)
+![DataGrid with CheckBoxColumn](Images\column-types\maui-datagrid-column-checkbox.png)
```

Comment:
What is the reason for removing the space here?

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/145664919/0ca3db29-1aed-49ca-ab46-0176ca7f3596), this is a continuous property, Hence I removed the space there.

Reply:
Please revert here. We need to describe the feature of the image here.

Reply:
Resolved the comments